test/integration/test_output_transform.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+import pytest
+
+from test.integration.utility import *
```

Comment:
Can we import specific modules or packages explicitly? It helps when debugging.

Reply:
Amended this in some places. But we may want to track this as tech debt in an issue, because this occurs in other places as well. I filed https://github.com/envoyproxy/nighthawk/issues/371 for that (as `tech-debt`)